plugins/hypervisors/xenserver/src/main/java/com/cloud/hypervisor/xenserver/resource/CitrixResourceBase.java

```diff
@@ -226,6 +228,8 @@ public String toString() {
     protected static final HashMap<VmPowerState, PowerState> s_powerStatesTable;
 
     public static final String XS_TOOLS_ISO_AFTER_70 = "guest-tools.iso";
+    public static final String BASEPATH = "/opt/xensource/packages/resources/";
```

Comment:
we might need to check/test if the path is the same across XS7.1 to xcp 8.2.

Reply:
I have verified this - `/opt/xensource/packages/` exists across the various versions of xenserver / xcp hosts , and the sub-directory resources, is created at the time of setting up the hosts